latex/Research_Strategy.tex

```diff
@@ -227,27 +233,29 @@
 %c. Mention uniqueness of SAMPL -- there are other predictive challenges (D3R, pKa coop, CASP) but none focused on driving quantitatively predictive protein-ligand modeling
 This work is also innovative because of the uniqueness of SAMPL.
 While there are other predictive challenges in the area of biomolecular modeling, such as D3R [ref], the pKa cooperative [ref], and CASP [ref],
+{\color{red}[JDC: Is CAPRI still a thing? Are there other docking or ligand binding prediction challenges?]}
```

Comment:
CAPRI seems to have [had an event in 2016](http://www.ebi.ac.uk/msd-srv/capri/) alongside CASP.